### PR TITLE
chore(tutorials): translate create_architecture.md

### PR DESCRIPTION
**Pull request by takam5f2**

### CODE REVIEW

File: docs/tutorials/create_architecture.md [hunk 1]
```diff
@@ -1,221 +1,225 @@
...
+
+An architecture file is a yaml-based file which has description of an application measured by CARET. In particular, an architecture file includes information representing the application structure, such as executors, nodes, callback groups, callbacks, topics, timers and so on. It can be used for another trial measurement because such information shall be same unless the application is changed.
 
 ---
 
-アーキテクチャファイルの主な役割は以下の通りです。
+User have to add the following description to architecture file.
+
+1. Target node chains and its structure
+2. Intra-communication of topic in a single node to calculate latency of the node
```
Comment (hsgwa) [inline docs/tutorials/create_architecture.md:21]: 1. static information (e.g. executors, nodes, callback)
2. latency definition

File: docs/tutorials/measurement.md [hunk 2]
```diff
@@ -1,105 +1,113 @@
...
 ```
 
-> 測定対象のアプリケーションも再ビルドが必要な理由
->
-> caret は rclcpp のヘッダーで読み込まれるテンプレート実装内にもトレースポイントを追加しています。  
-> トレースポイントを追加された rclcpp を有効にするため、測定対象のアプリケーションを再度ビルドする必要が有ります。  
-> 従って、 apt でインストールした demo_nodes_cpp などのアプリケーションは測定することが出来ません。  
-> 測定するにはソースファイルからビルドし直す必要があります。
+> Reason to build the target with CARET/rclcpp
+> Some tracepoints must be added to template implementation, which is referred by rclcpp, for CARET to measure performance.
```
Comment (hsgwa) [inline docs/tutorials/measurement.md:24]: which is referred by rclcpp header files

**Review by hsgwa — CHANGES_REQUESTED** [review]
I have written a comment with some corrections.
Please confirm.

File: docs/tutorials/create_architecture.md [hunk 3]
```diff
@@ -1,221 +1,225 @@
...
+           subscribe_topic_name: /topic2
+         - node_name: /timer_driven_node
+           publish_topic_name: /topic4
+           subscribe_topic_name: /topic3
+         - node_name: /actuator_dummy_node
+           publish_topic_name: UNDEFINED
+           subscribe_topic_name: /topic4
+   ```
 
 ## ノードレイテンシの定義指定
```
Comment (takam5f2) [inline docs/tutorials/create_architecture.md:138]: I left a sentence in Japanese, and I'll remove it.

File: docs/tutorials/create_architecture.md (hunk 1 above)
Comment (takam5f2) [reply docs/tutorials/create_architecture.md:21]: Here, I'd like to mention what users have to add in an architecture file. Static information and latency definition are main portion of the architecture file, but they are not added by users.

File: docs/tutorials/create_architecture.md [hunk 4]
```diff
@@ -1,221 +1,225 @@
-# 評価準備
+# Preparation for measuring latency of callback-chains
 
-CARET は、評価する際にアーキテクチャファイルの作成が必要です。
+CARET is capable of measuring latency of not only callback functions and topic communications, but also node chains.
```
[the remark this answers is not in the document]
Comment (takam5f2) [reply docs/tutorials/create_architecture.md:3]: I'd like to integrate callback duration the first of "Callback execution time".

File: docs/tutorials/measurement.md (hunk 2 above)
Comment (takam5f2) [reply docs/tutorials/measurement.md]: Fixed in https://github.com/tier4/CARET_doc/pull/36/commits/7331b7dbcec4edbb06e9a15fcba6806856201826

File: docs/tutorials/create_architecture.md (hunk 3 above)
Comment (takam5f2) [reply docs/tutorials/create_architecture.md]: fixed https://github.com/tier4/CARET_doc/pull/36/commits/8e5196debdafc6a3762d802af144337303de44ad

File: docs/tutorials/create_architecture.md (hunk 1 above)
Comment (takam5f2) [reply docs/tutorials/create_architecture.md]: Fixed in https://github.com/tier4/CARET_doc/pull/36/commits/d63db973ff51f0af897bef167dd4fe3ccab4c7c9

File: docs/tutorials/create_architecture.md [hunk 5]
```diff
@@ -1,221 +1,225 @@
...
 
-トレース結果からアーキテクチャファイルの雛形を作成できますが、一部手作業での修正が必要になります。  
-ここでは、アーキテクチャファイルの作成方法とその修正方法を説明します。
+## Overview of an architecture file
 
-## アーキテクチャファイルとは
+To deal with such difficulty of specifying node chains, CARET serves configuration for users to choose target node chains. The configuration is defined in a yaml file, which is called "**architecture file**". An architecture file describes a target application's structure. CARET can create an architecture file for minimum description automatically with using measured data, but it does not have target node chains. You will add description for target node chains to a generated architecture file.
```
[the remark this answers is not in the document]
Comment (takam5f2) [reply docs/tutorials/create_architecture.md]: Fixed in https://github.com/tier4/CARET_doc/pull/36/commits/d63db973ff51f0af897bef167dd4fe3ccab4c7c9

File: docs/tutorials/create_architecture.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (takam5f2) [reply docs/tutorials/create_architecture.md]: Fixed in https://github.com/tier4/CARET_doc/pull/36/commits/d63db973ff51f0af897bef167dd4fe3ccab4c7c9

File: docs/tutorials/create_architecture.md [hunk 6]
```diff
@@ -1,221 +1,221 @@
...
 
-トレース結果からアーキテクチャファイルの雛形を作成できますが、一部手作業での修正が必要になります。  
-ここでは、アーキテクチャファイルの作成方法とその修正方法を説明します。
+- callback latency
+- communication latency (inter-node)
+- node latency (intra-node)
+- path latency (intra-node & inter-node)
 
-## アーキテクチャファイルとは
+The following figure shows definition of node latency and path latency.
```
Comment (hsgwa) [inline docs/tutorials/create_architecture.md:10]: Insert line breaks between figures and text.
> The following figure shows definition of node latency and path latency.
>
> [figure]

**Review by hsgwa — CHANGES_REQUESTED** [review]
Correct only the line breaks.

File: docs/tutorials/create_architecture.md (hunk 6 above)
Comment (takam5f2) [reply docs/tutorials/create_architecture.md:10]: @hsgwa  fixed in https://github.com/tier4/CARET_doc/pull/36/commits/25526e73df6e953f6cc1130fec348ec9cffc2963

**Comment by takam5f2** [comment]
@hsgwa  
I resolved conflict between branched with paying attention to https://github.com/tier4/CARET_doc/pull/39
Please check if the latest version of this branch can be merged main branch.

Best

**Review by hsgwa — APPROVED** [review]
LGTM

**Pull request merged**